source/expressions.tex

```diff
@@ -235,9 +235,9 @@
 
 \pnum
 Whenever a glvalue appears as an operand of an operator that
-expects a prvalue for that operand, the
+expects a prvalue for that operand, an
```

Comment:
Reading all the "expected to be" phrasings below makes me want to ask for "requires" here.
Example: "that requires a prvalue for that operand..." or "The operand is required to be a prvalue."
The "required" conveys better that, if you can't make it so, you're toast. (Just a slight wording preference, not a strong feeling.)